docs/migration_guide.rst

```diff
@@ -0,0 +1,47 @@
+Migration from pysat 2
+======================
+
+With the release of pysat 3.0.0, the pysat project now keeps instrument modules
+within distinct packages. Each of these packages acts as an interface between
+the core pysat package and a unique data provider.  pysatNASA fills this role
+for the `Space Physics Data Facility <https://spdf.gsfc.nasa.gov/>`_ at NASA.
+
+Registering the pysatNASA library
+---------------------------------
+
+While each module can be loaded separately, users of pysat 2 may find it easier
```

Comment:
I don't think of registering the instruments as limited to pysat2 users. It does maximize backwards compatibility with pysat2 code though.

Reply:
updated